### ISSUE TITLE
Modify reading order on asset show page

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#1791]
As a screen reader user, I want the page title and information to be read to me before I interact with the media on the asset page.

Currently, a screen reader user navigating through the asset page will encounter the asset page media _before_ the asset title and other information is read to them.

![screen shot 2018-05-04 at 7 59 35 am](https://user-images.githubusercontent.com/1686111/39626853-b093c06e-4f71-11e8-874e-c4cd6a71477a.png)

On the asset show page `/app/views/hyrax/file_sets/show.html.erb` ([link](https://github.com/mlibrary/heliotrope/blob/master/app/views/hyrax/file_sets/show.html.erb)):

- [ ] Keep the visual display the same
- [ ] [Modify the HTML starting at line 34](https://github.com/mlibrary/heliotrope/blob/master/app/views/hyrax/file_sets/show.html.erb#L34) so the page title and asset info is first, media is second
- [ ] Using [column ordering](https://getbootstrap.com/docs/3.3/css/#grid-column-ordering), modify the CSS so that the visual display remains the as is.
- [ ] Consider adding a navigation/skip link to the media after the title?
- [ ] When read with a screen reader, the asset title and information should be presented to the user first.
